### DISCUSSION TITLE
how to deploy the repository

### DISCUSSION THREAD

**Discussion by sarah127** [autogluon/autogluon#1135] [Q&A]
Do I need to host a server on AWS to run and test the repository ? Is there a way to deploy it locally or using vscode server?
can I use colab to execute the code in the repository?

**Comment by talhaanwarch** [comment]
yes, you can use it locally or even on colab.

**Reply by sarah127** [reply]
How, the files in the repository seem not connected, even when I installed each library independently.

**Reply by talhaanwarch** [reply]
can you share your code to look into it

**Comment by Innixma** [comment]
Can you try following the installation documentation to build from source?

https://auto.gluon.ai/stable/install.html

```
python3 -m pip install -U pip
python3 -m pip install -U setuptools wheel
python3 -m pip install -U "mxnet<2.0.0"
git clone https://github.com/awslabs/autogluon
cd autogluon && ./full_install.sh
```

We also have a Colab tutorial:

https://colab.research.google.com/drive/1oT3zNsj9et8s1bJNx7VeHOn_mfpBoe4q?usp=sharing

**Reply by sarah127** [reply]
thank you, but I met modifying the source code not using the package

**Reply by Innixma** [reply]
Building from source allows you to modify the source code. This is how I develop AutoGluon (using the commands I provided). I can then load the project into PyCharm and edit from there.
